docs/reference/data-streams/lifecycle/tutorial-manage-data-stream-retention.asciidoc

```diff
@@ -0,0 +1,184 @@
+[[tutorial-manage-data-stream-retention]]
+=== Tutorial: Data stream retention
+
+preview::[]
+
+In this tutorial, we are going to go over the data stream lifecycle retention, define it, go over how it can be configured and how
+it can be applied. Keep in mind, the following options apply only to data streams that have are managed by the data stream lifecycle.
+
+
+////
+[source,console]
+----
+PUT /_index_template/template
+{
+  "index_patterns": ["my-data-stream*"],
+  "template": {
+    "lifecycle": {}
+  },
+  "data_stream": { }
+}
+
+PUT /_data_stream/my-data-stream
+----
+// TESTSETUP
+////
+
+////
+[source,console]
+----
+DELETE /_data_stream/my-data-stream*
+DELETE /_index_template/template
+DELETE /_data_stream/_global_retention
+----
+// TEARDOWN
+////
+
+You can verify this using the <<data-streams-get-lifecycle,get data stream lifecycle API>>:
+
+[source,console]
+--------------------------------------------------
+GET _data_stream/my-data-stream/_lifecycle
+--------------------------------------------------
+
+The result should look like this:
+
+[source,console-result]
+--------------------------------------------------
+{
+  "data_streams": [
+    {
+      "name": "my-data-stream",                                   <1>
+      "lifecycle": {
+        "enabled": true                                           <2>
+      }
+    }
+  ]
+}
+--------------------------------------------------
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+<1> The name of your data stream.
+<2> Ensure that the lifecycle is enabled, meaning this should be `true`.
+
+. <<what-is-retention>>
+. <<retention-configuration>>
+. <<effective-retention-calculation>>
+. <<effective-retention-application>>
+
+[discrete]
+[[what-is-retention]]
+==== What is data stream retention?
+
+We define retention as the least amount of time the data of a data stream are going to be kept in {es}. After this time period
+has passed, {es} is allowed to remove this data to free up space and/or manage costs.
+
+NOTE: Retention does not define the period that the data will be removed, but the minimum time period they will be kept.
+
+We define 4 different types of retention:
+
+* The data stream retention, or `data_retention`, which is the retention configured on the data stream level. It can be
+set via an <<index-templates,index template>> for future data streams or via the <<data-streams-put-lifecycle, set data
+stream lifecycle API>> for an existing data stream. When the data stream retention is not set, it implies that the data
+needs to be kept forever.
+* The global default retention, or `default_retention`, which is a retention configured on a project level and will be
+applied to all data streams managed by data stream lifecycle that do not have `data_retention` configured. Effectively,
+it ensures that there will be no data streams keeping their data forever. This can be set via the
+<<data-streams-put-global-retention, set global retention API>>.
```

Comment:
You call it the `PUT global retention API` in the next paragraph. And when you go to that page, that page calls itself the `Updates the global retention of data streams`

Reply:
I will consistently use PUT because that's what we do for the lifecycle as well.